src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api.Tests/Controllers/PublicationsControllerTests.cs

```diff
@@ -356,6 +395,323 @@ private static async Task<HttpResponseMessage> GetPublication(HttpClient client,
+                .WithStatusPublished()
+                .WithPublicationId(publicationId)
+                .Generate();
+
+            await TestApp.AddTestData<PublicDataDbContext>(context => context.DataSets.Add(dataSet));
+
+            var dataSetVersion = DataFixture
+                .DefaultDataSetVersion(
+                    filters: 1,
+                    indicators: 1,
+                    locations: 1,
+                    timePeriods: 3)
+                .WithStatusPublished()
+                .WithDataSet(dataSet)
+                .FinishWith(dsv => dataSet.LatestVersion = dsv)
+                .Generate();
+
+            await TestApp.AddTestData<PublicDataDbContext>(context =>
+            {
+                context.DataSetVersions.Add(dataSetVersion);
+                context.DataSets.Update(dataSet);
+            });
+
+            var client = BuildApp().CreateClient();
+
+            var response = await ListDataSets(
+                client: client,
+                publicationId: publicationId,
+                page: 1,
+                pageSize: 1);
+
+            var content = response.AssertOk<DataSetPaginatedListViewModel>(useSystemJson: true);
+
+            Assert.NotNull(content);
+            Assert.Equal(1, content.Paging.Page);
+            Assert.Equal(1, content.Paging.PageSize);
+            Assert.Equal(1, content.Paging.TotalResults);
+
+            var result = Assert.Single(content.Results);
+
+            Assert.Equal(dataSet.Id, result.Id);
+            Assert.Equal(dataSet.Title, result.Title);
+            Assert.Equal(dataSet.Summary, result.Summary);
+            Assert.Equal(dataSet.Status, result.Status);
+            Assert.Equal(dataSet.SupersedingDataSetId, result.SupersedingDataSetId);
+            Assert.Equal(dataSetVersion.Version(), result.LatestVersion.Number);
+            Assert.True(Math.Abs(dataSetVersion.Published!.Value.Subtract(result.LatestVersion.Published).TotalMilliseconds) < 1);
+            Assert.Equal(dataSetVersion.TotalResults, result.LatestVersion.TotalResults);
+            Assert.Equal(
+                TimePeriodFormatter.Format(
+                    dataSetVersion.MetaSummary.TimePeriodRange.Start.Year,
+                    dataSetVersion.MetaSummary.TimePeriodRange.Start.Code),
+                result.LatestVersion.TimePeriods.Start);
+            Assert.Equal(
+                TimePeriodFormatter.Format(
+                    dataSetVersion.MetaSummary.TimePeriodRange.End.Year,
+                    dataSetVersion.MetaSummary.TimePeriodRange.End.Code),
+                result.LatestVersion.TimePeriods.End);
+            Assert.Equal(dataSetVersion.MetaSummary.GeographicLevels.Order(), result.LatestVersion.GeographicLevels.Order());
+            Assert.Equal(dataSetVersion.MetaSummary.Filters.Order(), result.LatestVersion.Filters.Order());
+            Assert.Equal(dataSetVersion.MetaSummary.Indicators.Order(), result.LatestVersion.Indicators.Order());
```

Comment:
it was because technically I don't care if the order is different, only if they contain the same items. So if the implementation ever changed and they were in a different order for whatever reason, then this test would still pass because it doesn't care.

However, I've removed the `Order` calls for brevity